theano/gpuarray/opt.py

```diff
@@ -1990,43 +1991,104 @@ def local_inplace_gpu_solve(node):
 
 
 # Cholesky decomposition
-@register_opt('fast_compile')
-@op_lifter([slinalg.Cholesky])
-@register_opt2([theano.tensor.slinalg.Cholesky], 'fast_compile')
 def local_gpu_cholesky(op, context_name, inputs, outputs):
```

Comment:
Now I understand. You want to prioritize the cusolver op over the magma op. A quick comment would be great. It seem good.

But I taught magma was faster in your case. Is that exact? Or it was for other magma ops? Add a note in the comment why you put cusolver first and make sure to put as the first the one you would suggest as the best default.

thanks